app/views/interviewee/_show.html.erb

```diff
@@ -7,8 +7,9 @@
   <%= react_component('MediaPlayer',
     id: parsed_children.try(:first).try(:[], "id_t").try(:gsub, '/', '-'),
     src: parsed_children.try(:first).try(:[], "url_t"),
-    peaks: parsed_children.try(:first).try(:[], "peaks"), image: "/avatar.jpg"),
-    typeOfResource: type_of_resource(document)
+    peaks: parsed_children.try(:first).try(:[], "peaks"),
```

Comment:
Brakeman Audit:
Error: app/views/interviewee/_show.html.erb:10 :: parse error on value "," (tCOMMA) Could not parse app/views/interviewee/_show.html.erb
Location: Could not parse /Users/aprilrieger/softserv/oral-history/app/views/interviewee/_show.html.erb

Reply:
did you see of this broke something in the UI? We are adding image and typeOfResource here for a reason.

Reply:
oh wait, I see it below here. sorry